src/animations/tweens.js

```diff
@@ -0,0 +1,54 @@
+import gsap from 'gsap';
+
+export const buttonAppearTween = button =>
+  gsap.to(button, {
+    autoAlpha: 1,
+    duration: 2,
+  });
+
+export const dropButtonTween = button =>
+  gsap.to(button, {
+    y: '+=220',
+    ease: 'power4.out',
+    duration: 1,
+  });
+
+export const dropItemTween = item =>
+  gsap.fromTo(
+    item,
+    { y: '-=300', autoAlpha: 0 },
+    {
+      y: '+=300',
+      duration: 1,
+      ease: 'bounce.out',
+      autoAlpha: 1,
+    }
+  );
+
+export const enterSpanFromLeftTween = span =>
```

Comment:
as `button` is quite generic and special (there is not to much similar to buttons html tags), the question is - if there is something specific here only for span? There is no chance to use this animation (extracted to separate place) on `div` or something else?